src/web/app/src/components/DependenciesPage.tsx

```diff
@@ -0,0 +1,60 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { useEffect, useState } from 'react';
+
+import DependenciesTable from './DependenciesTable';
+import { dependencyDiscoveryUrl } from '../config';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.background.default,
+    fontFamily: 'Spartan',
+    padding: '1em 0 2em 0',
+    paddingTop: 'env(safe-area-inset-top)',
+    wordWrap: 'break-word',
+    [theme.breakpoints.down(1024)]: {
+      maxWidth: 'none',
+    },
+    '& h1': {
+      color: theme.palette.text.secondary,
+      fontSize: 24,
+      transition: 'color 1s',
+      marginTop: 0,
+    },
+  },
+  container: {
+    padding: '2vh 18vw',
+    [theme.breakpoints.down(1024)]: {
+      padding: '2vh 8vw',
+      wordWrap: 'break-word',
+    },
+  },
+}));
+
+const DependenciesPage = () => {
+  const classes = useStyles();
+  const [dependencies, setDependencies] = useState<string[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const fetchDependenciesData = await fetch(`${dependencyDiscoveryUrl}/projects`);
+
+        // Shuffle the list of dependencies
```

Comment:
Why not alphabetize?

Reply:
Oh because, I find it weird to show all packages starting `@` in the first 3-4 pages, but I can remove that.